assembl/static2/js/app/components/debate/common/themes.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Translate } from 'react-redux-i18n';
+import { Grid, Row, Col } from 'react-bootstrap';
+import { isPhaseStarted, getStartDatePhase } from '../../../utils/timeline';
+import NotStartedPhase from './notStartedPhase';
+import ThematicPreview from '../../common/thematicPreview';
+
+class Themes extends React.Component {
+  render() {
+    const { thematics, identifier } = this.props;
+    const { debateData } = this.props.debate;
+    const { rootPath } = this.props.context;
+    const phaseStarted = isPhaseStarted(debateData.timeline, identifier);
+    const startDatePhase = getStartDatePhase(debateData.timeline, identifier);
+    if (phaseStarted) {
+      return (
+        <section className="themes-section">
+          <Grid fluid className="background-grey">
+            <div className="max-container">
+              <div className="title-section">
+                <div className="title-hyphen">&nbsp;</div>
+                <h1 className="dark-title-1">
+                  <Translate value="debate.survey.themesTitle" />
+                </h1>
+              </div>
+              <div className="content-section">
+                <Row className="no-margin">
+                  {thematics.map((thematic, index) => {
+                    return (
+                      <Col xs={12} sm={6} md={3} className={index % 4 === 0 ? 'theme no-padding clear' : 'theme no-padding'} key={index}>
```

Comment:
Mouhahaha... Help me god to explain in english. This is a rule define by the design (aka Roxana). The behavior is different according to the number of thumbnails. It works with a multiple of 4.